tools/generators/move-packages/index.spec.ts

```diff
@@ -130,6 +133,14 @@ describe('move-packages generator', () => {
       const storybookMainJsFile = tree.read(storybookMainJsPath, 'utf8') as string;
       expect(storybookMainJsFile.split(' ').filter(s => s.includes(newPath))).toHaveLength(2);
     });
+    it('should update the CODEOWNERS file with the new relative path', async () => {
+      await generator(tree, options);
+      const newPath = `packages/${options.destination}`;
+      const oldPath = `packages/test`;
+      const codeOwnersFile = tree.read(joinPathFragments('/.github', 'CODEOWNERS'), 'utf8') as string;
+      expect(codeOwnersFile.split(' ').filter(s => s === oldPath)).toHaveLength(0);
```

Comment:
**suggestion:**

can we simplify this test/make it more obvious ? something like following:

```ts
const oldOwnerDeclaration = `packages/test  @dummyOwner`;
const newOwnerDeclaration = `packages/${options.destination}  @dummyOwner`;

let codeOwnersFile = tree.read(joinPathFragments('/.github', 'CODEOWNERS'), 'utf8');
expect(codeOwnersFile.indexOf(oldOwnerDeclaration) !== -1).toBeTrue();

await generator(tree, options);

codeOwnersFile = tree.read(joinPathFragments('/.github', 'CODEOWNERS'), 'utf8');
expect(codeOwnersFile.indexOf(oldOwnerDeclaration) !== -1).toBeFalse();
expect(codeOwnersFile.indexOf(newOwnerDeclaration) !== -1).toBeTrue();
```

Reply:
Done in [e732115](https://github.com/microsoft/fluentui/pull/22624/commits/e732115e91537fdafe671341f4d475eeeb16db82)